docs/reference/changelog.rst

```diff
@@ -4,20 +4,22 @@
 Changelog
 *********
 
-5.5.0 (2025-MM-DD)
+5.5.0 (2025-07-17)
 ------------------
 
 Services
 ========
 
 - Add a new State service that manages a global state between manager and managed
   instances of an application.
+- Make the Project service compatible with multi-base platform definitions
```

Comment:
```suggestion
- Make the Project service compatible with multi-base platform definitions.
```

Nitpick of nitpicks, but you added a sentence with the point down in the page...